sql/19_route.up.sql

```diff
@@ -0,0 +1,21 @@
+CREATE TABLE `route`
+(
+    `id`                 varchar(35)         NOT NULL,
+    `name`               varchar(100)        NOT NULL,
+    `description`        varchar(200)        NOT NULL,
+    `distance_meters`    int unsigned        NOT NULL,
+    `duration_seconds`   int unsigned        NOT NULL,
+    `start_poi`          varchar(35)         NOT NULL,
+    `end_poi`            varchar(35)         NOT NULL,
+    `image`              varchar(200)        NOT NULL,
+    `image_border_color` varchar(10)         NOT NULL,
+    `reversible`         tinyint(1) unsigned NOT NULL,
+    `tags`               text DEFAULT NULL,
+    `type`               tinyint unsigned    NOT NULL,
+    `updated`            int unsigned        NOT NULL,
+    `version`            int unsigned        NOT NULL,
+    `waypoints`          text                NOT NULL,
```

Comment:
Should `waypoints` and `tags` use `json` type instead of `text`? (I guess other tables use `text` too. Just wondering.)

Reply:
Unfortunately not all sql drivers support the json type. Since we're not querying directly into the json this shouldn't matter.

Reply:
Also, on Mariadb the JSON type is just an alias for `LONGTEXT`. So it doesn't really matter.